hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTimeline.java

```diff
@@ -230,6 +230,14 @@ static boolean compareTimestamps(String commit1, String commit2, BiPredicate<Str
     return predicateToApply.test(commit1, commit2);
   }
 
+  /**
+   * Return true if specified timestamp is in range (startTs, endTs].
+   */
+  static boolean isInRange(String timestamp, String startTs, String endTs) {
```

Comment:
The HoodieDefaultTimeline has implementation -> `public HoodieDefaultTimeline findInstantsInRange(String startTs, String endTs)` already, can we try to reuse that ?

Reply:
HoodieDefaultTimeline has changes to use this method. I refactored to make this part reusable

`  @Override
  public HoodieDefaultTimeline findInstantsInRange(String startTs, String endTs) {
    return new HoodieDefaultTimeline(
        instants.stream().filter(s -> HoodieTimeline.isInRange(s.getTimestamp(), startTs, endTs)),
        details);
  }
`

Reply:
makes sense